smileidentity/web-client
# redesign: remove `previewBVNMFA`, implicitly switch from `BVN` to `BVN_MFA`

## Test Steps
- visit the [demo link](https://sid-launch-bvn-mfa.glitch.me)
- execute a bank verification job (Basic / Enhanced / Biometric KYC)
- ensure that it's submitted
- visit the portal for partner id 1825
- verify that the IDs show up as `BVN_MFA`
- verify that there's a loading spinner when you click the button "Continue" or "Submit" buttons


## Deployment Steps
- [ ] Merge to main
- [ ] Remove `id_number` from the `required_fields` list in DynamoDB table `smile-actions`. it is nested in the following levels, from purpose `web_app.hosted_web.{basic,enhanced,biometric}_kyc.id_types.NG.BVN.required_fields`
- [ ] run a cache invalidation in the cdn.smileidentity.com distribution on cloudfront (optional, but reduces time to cdn propagation)

## Post Deploy Steps
- [ ] Run BVN verification Basic, Enhanced, Biometric KYC jobs
